unit/simple_dsl_spec.cr

```diff
@@ -0,0 +1,418 @@
+require "./spec_helper"
```

Comment:
This is tested by stdlib's `spec`. It makes it easier to understand what is happening, when DSL itself breaks.